src/components/List/components/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './ListItem';
+export * from './SimpleContainer';
```

Comment:
Do we need SimpleContainer as exported component? It is not documented anywhere

Reply:
Actually we don`t need